Fix choreo-connect documentation (#3767)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-web-socket-api.md

# Deploy WebSocket API in Choreo Connect

> A Web Socket API can be deployed in Choreo Connect via API Manager.

Reviewer: ```suggestion
A WebSocket API can be deployed in Choreo Connect via API Manager.
```

> Before You begin

Reviewer: Before You begin section does not have any steps.

Author: Fixed in https://github.com/wso2/docs-apim/pull/3767/commits/03ed857c6384c8187453777bd2b9f1d390ca44d4

### Step 1 - Create a WebSocket API in API Manager

>  Please follow [Create a WebSocket API]({{base_path}}/design/create-api/create-streaming-api/create-a-websocket-streaming-api/) and create the WebSocket API.

Reviewer: ```suggestion
 For instructions on how to create a WebSocket API, see [Create a WebSocket API]({{base_path}}/design/create-api/create-streaming-api/create-a-websocket-streaming-api/).
```

> ### Step 2 - Deploy the API in Choreo Connect environment.

Reviewer: ```suggestion
### Step 2 - Deploy the API in the Choreo Connect environment
```

> Deploy the API in Choreo Connect by following the [Deploy API]({{base_path}}/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-an-api/)

Reviewer: ```suggestion
For more information on deploying the API in Choreo Connect, see [Deploy API]({{base_path}}/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-an-api/).
```

### Step 3 - Generate an Access Token to invoke the API

> By default, the WebSocket API is protected by Oauth2 token.

Reviewer: ```suggestion
By default, the WebSocket API is protected by an OAuth2 token.
```

> Generate a JWT Access token by following, [Get a Test Key to Invoke an API]({{base_path}}/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-the-integrated-api-console/#get-a-test-key-to-invoke-an-api)

Reviewer: ```suggestion
For more information on generating a JWT Access token, see [Get a Test Key to Invoke an API]({{base_path}}/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-the-integrated-api-console/#get-a-test-key-to-invoke-an-api).
```

### Invoke the API using a WebSocket client

> The websocket api exposed via Choreo Connect, can be invoked by using a WebSocket client.

Reviewer: ```suggestion
The WebSocket API exposed via Choreo Connect can be invoked by using a WebSocket client.
```

The JWT token should be set as the Authorization header in the initial WebSocket handshake request.

!!!note

>     The same ports 9095 (https) and 9090 (http) are used for WebSocket APIs.

Reviewer: ```suggestion
    The same ports 9095 (HTTPS) and 9090 (HTTP) are used for WebSocket APIs.
```

> Please follow [Step 4 - Invoke the WebSocket API]({{base_path}}/tutorials/streaming-api/create-and-publish-websocket-api/#step-4-invoke-the-websocket-api) on how to invoke a WebSocket API.

Reviewer: ```suggestion
Invoke the WebSocket API by carrying out [Step 4]({{base_path}}/tutorials/streaming-api/create-and-publish-websocket-api/#step-4-invoke-the-websocket-api) in the Create and Publish a WebSocket API tutorial.
```
